packages/client/helpers/build.ts

```diff
@@ -90,6 +90,8 @@ function bundleTypeDefinitions(filename: string, outfile: string) {
         '@prisma/internals',
         '@prisma/engine-core',
         '@prisma/generator-helper',
+        '@prisma/debug',
+        '@opentelemetry/api',
```

Comment:
This is definitely expected for `@prisma/debug` and I'm sure adding it here it correct, but I'm not so sure about the `@opentelemetry/api`.

@millsp is this the right way to fix this?